Makefile

```diff
@@ -315,6 +317,12 @@ EBS_CSI_DRIVER_DIR=./ecs-agent/daemonimages/csidriver
 ebs-csi-driver:
 	$(MAKE) -C $(EBS_CSI_DRIVER_DIR) $(MFLAGS) bin/ebs-csi-driver
 
+
+ebs-csi-driver-tar:
```

Comment:
Is there use of `ebs-csi-driver` command, and if it can be safe to remove?

Otherwise, can we add a comment/description for the commands - `ebs-csi-driver` and `ebs-csi-driver-tar` to easily differentiate what purpose they serve?